assets/scripts/ui/__tests__/KeyboardKey.test.js

```diff
@@ -1,25 +1,42 @@
 /* eslint-env jest */
 import React from 'react'
-import { shallow } from 'enzyme'
+import { cleanup } from '@testing-library/react'
 import KeyboardKey from '../KeyboardKey'
+import { renderWithReduxAndIntl } from '../../../../test/helpers/render'
 
 describe('KeyboardKey', () => {
+  afterEach(() => {
+    cleanup()
+  })
```

Comment:
According to the changelog for react-testing-library@9.x (https://github.com/testing-library/react-testing-library/releases/tag/v9.0.0), the cleanup now happens automatically after every test. We no longer need to include this manually.